lib/rucio/db/sqla/constants.py

```diff
@@ -1,19 +1,27 @@
-# Copyright European Organization for Nuclear Research (CERN)
+# -*- coding: utf-8 -*-
+# Copyright 2015-2020 CERN
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
-# You may not use this file except in compliance with the License.
-# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 #
 # Authors:
-# - Vincent Garonne, <vincent.garonne@cern.ch>, 2013-2017
-# - Mario Lassnig, <mario.lassnig@cern.ch>, 2014, 2017
-# - Martin Barisits, <martin.barisits@cern.ch>, 2014-2019
-# - Cedric Serfon, <cedric.serfon@cern.ch>, 2015-2018
-# - Wen Guan, <wen.guan>, 2016
+# - Vincent Garonne <vincent.garonne@cern.ch>, 2015-2017
+# - Wen Guan <wen.guan@cern.ch>, 2015-2016
+# - Cedric Serfon <cedric.serfon@cern.ch>, 2016
```

Comment:
Why do you change the dates ? Is it based on git blame ?

Reply:
its based on tools/add_header and therefore the git history - is there a problem with these lines? The history isn't always correct, especially when files were moved around

Reply:
The constants file was probably (content) copied once when we move to the sqla folder. Sometimes the dates in these files are also wrongly inserted by people. I think it is fine to just rely on the add_header one.